lib/core/test/unit/Cardano/WalletSpec.hs

```diff
@@ -420,7 +420,11 @@ setupFixture (wid, wname, wstate) = do
 
     block0Date = StartTime $ posixSecondsToUTCTime 0
 
+    k :: Quantity "block" Word32
+    k = Quantity 2160
```

Comment:
This stability constant (`2160`) appears to be defined in a few places. Could it be defined in a single place and imported?

Also, would it make the code more readable if we call it `epochStability` or something similar? (I did happen to know what `k` is, but perhaps others reading the code might not?)

Reply:
> Could it be defined in a single place and imported?

Yes and no. It would be wrong to hard-code it in one place (although it is in `http-bridge@.../Compatibility.hs`. Here in this test scenario (like in all scenarios where it's hard-coded), its actual value doesn't matter. Could be 42, could be 87541. Force of the habit and lack of originality made us choose 2160. Perhaps an `Arbitrary` would better convey the idea.

Reply:
*I've changed the declaration of `BlockchainParameters` to use named record fields which removes the need for intermediate variable and make things more readable:

```hs
    bp = BlockchainParameters
        { getGenesisBlock = block0
        , getGenesisBlockDate = StartTime $ posixSecondsToUTCTime 0
        , getFeePolicy = LinearFee (Quantity 14) (Quantity 42)
        , getSlotLength = SlotLength 1
        , getEpochLength = EpochLength 21600
        , getTxMaxSize = Quantity 8192
        , getEpochStability = Quantity 2160
        }
```

Reply:
> It would be wrong to hard-code it in one place

Why? If we intend it to be the same value in each of these places, then why is it wrong to just write it once and then re-import it each time?

Of course, if it's an arbitrary value, then why not just generate arbitrary values when needed?